src/github.com/t3chguy/matrix-static/matrix-static.go

```diff
@@ -266,7 +274,7 @@ func main() {
 				AtBottomEnd: jobResult.AtBottomEnd,
 
 				Sanitizer:         sanitizerFn,
-				HomeserverBaseURL: client.HomeserverURL.String(),
+				HomeserverBaseURL: config.MediaBaseURL,
```

Comment:
surely "HomeserverBaseURL" should still be the HomeserverURL ?

Reply:
Well this is the URL being passed to the HTML template, as far as the end-users are concerned they should only ever see the MediaBaseURL because the HomeserverURL may not resolve for them, as is in the case of view.matrix.org